src/login.rs

```diff
@@ -73,3 +73,28 @@ pub fn get_cgroup(pid: Option<pid_t>) -> Result<String> {
     let cg = unsafe { MString::from_raw(c_cgroup) };
     Ok(cg.unwrap().to_string())
 }
+
+/// Determines the session identifier of a process.
+///
+/// Specific processes can be optionally targeted via their PID. When no PID is
+/// specified, operation is executed for the calling process.
+/// This method can be used to retrieve a session identifier.
+pub fn get_session(pid: Option<pid_t>) -> Result<String> {
+    let mut c_session: *mut c_char = ptr::null_mut();
+    let p: pid_t = pid.unwrap_or(0);
```

Comment:
@jmesmon Yeah, good point. I'm also wondering if `Option<>` in an input parameter is idiomatic at all. Not sure as I'm just a newbie at rust.
OTOH for now I think we should keep it as-is for consistency with other functions like `get_unit()`. Anyway I agree that it's a thing to be changed with a major release.